src/Sushi.Mediakiwi.Headless/ScriptTagHelperComponent.cs

```diff
@@ -19,6 +19,40 @@ public class ScriptTagHelperComponent : TagHelperComponent
         [ViewContext]
         public ViewContext ViewContext { get; set; }
 
+        private void PerformReplacements(TrackingScriptSimple script)
+        {
+            if (script?.ReplacementItems?.Count > 0)
+            {
+                // Are all marked as mandatory ?
+                var allMandatory = script.ReplacementItems.Count == script.ReplacementItems.Count(x => x.MustBeFilled);
```

Comment:
Use script.ReplacementItems.All(x => x.MustBeFilled) for better performance

Reply:
Agreed, done